### PR TITLE
added Cnn test

### PR DESCRIPTION
**Pull request by bare612**

### CODE REVIEW

**Comment by bare612** [comment]
Cnn is ready

File: Cnn/src/test/java/home/TestNews.java [hunk 1]
```diff
@@ -1,22 +1,180 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.util.List;
+
 public class TestNews extends CommonAPI {
-@Test
```
Comment (selmanHa) [inline Cnn/src/test/java/home/TestNews.java]: Abdi, is this by mistake?

File: Cnn/src/test/java/home/TestNews.java [hunk 2]
```diff
@@ -1,22 +1,180 @@
...
-    Assert.assertEquals(actualTitle,title);
+    @Test
+    public void test8() {
+        driver.findElement(By.name("health"));
+        driver.findElement(By.cssSelector(".nav-linksstyles__Link-sc-1tike8v-0")).click();
 
     }
+
     @Test
-    public void test2(){
```
Comment (selmanHa) [inline Cnn/src/test/java/home/TestNews.java]: I think you already have test2()

File: Cnn/src/test/java/home/TestNews.java [hunk 3]
```diff
@@ -1,22 +1,180 @@
...
+    public void test14() {
+        driver.findElement(By.linkText("LIVE UPDATES: Impeachment inquiry")).click();
+    }
+
+    @Test
+    public void test15() {
+        driver.findElement(By.linkText("Boris Johnson")).click();
     }
 
-}
```
Comment (selmanHa) [inline Cnn/src/test/java/home/TestNews.java]: This closing brace is may be from wrongly added test2() method

**Pull request merged**